lighthouse-core/report/v2/renderer/dom.js

```diff
@@ -127,6 +127,28 @@ class DOM {
   }
 
   /**
+   * @param {string} text
+   * @return {!Element}
+   */
+  convertMarkdownCodeSnippets(text) {
```

Comment:
Not sure that's easy to do. They're different regexs and construct different dom. We might be able to combine, but following the code would be really messy.

Renamed.